homeassistant/components/sonos/media_player.py

```diff
@@ -1237,4 +1247,7 @@ def device_state_attributes(self):
         if self._speech_enhance is not None:
             attributes[ATTR_SPEECH_ENHANCE] = self._speech_enhance
 
+        if self._status_light is not None:
+            attributes[ATTR_STATUS_LIGHT] = self._status_light
```

Comment:
Can you describe a use for this attribute? We shouldn't stuff everything into the state attributes just because we can.

Reply:
Hi @amelchio 

Thanks for taking the time to review the pull request.

We have Sonos players in our bedrooms, as well as in our living room where guests stay (on the sofa bed) and would like to reduce the "light pollution" in the house in the evening/night. 

The ability to control the Status Light via HomeAssistant allows us to automate this at night/bed time.

Hope this can be included in the integration

Thanks

Ian

Reply:
Right, toggling the light is fine and your code looks good.

I am just asking about the need to read out the current state of the status light? I don't see anyone creating an automation based on that ...

Reply:
Ah, I see. Well, I’m easy either way on that, I was just try to keep it in line with the other set_option attributes.

Reply:
Yeah, I also do like consistency but let's anyway skip it for now. If needed, we can always add it later but removing stuff is much harder.

Reply:
Thanks, this made me think.

We were actually wrong in adding this as an attribute in the first place. The status light should have been a separate light entity so the normal light UI could be used to turn it on and off. I think that is the way forward, not adding another attribute.

Also, the night/speech settings should be switch entities that could likewise be controlled separately without everybody having to add template switches.

Reply:
> Thanks, this made me think.
> 
> We were actually wrong in adding this as an attribute in the first place. The status light should have been a separate light entity so the normal light UI could be used to turn it on and off. I think that is the way forward, not adding another attribute.
> 
> Also, the night/speech settings should be switch entities that could likewise be controlled separately without everybody having to add template switches.

That makes sense, maybe Sonos app can be refered to.